libavfilter/dnn_filter_common.c

```diff
@@ -106,18 +106,18 @@ int ff_dnn_set_classify_post_proc(DnnContext *ctx, ClassifyPostProc post_proc)
     return 0;
 }
 
-DNNReturnType ff_dnn_get_input(DnnContext *ctx, DNNData *input)
+int ff_dnn_get_input(DnnContext *ctx, DNNData *input)
```

Comment:
will there be build warning since ctx->model->get_input still returns DNNReturnType?

anyway, i think this is a good idea to decouple the change.

Reply:
I didn't see any build warning related to `libavfilter` after this commit (no warning earlier also).

Decouple the change means?

Reply:
i mean to separate the changes into small commits.

Reply:
Oh, got it! Currently, I have migrated one module per commit which is a bigger change. Migrating one function each commit instead will lead to smaller commits.

Reply:
Oh, okay. :smile: